### DISCUSSION TITLE
카카오맵에서 매크로 실행시 무조건 네이버 예약페이지로 이동됩니다.

### DISCUSSION THREAD

**Discussion by SingleBungleTeemo** [changdoc/naver-vaccine-macro#262] [Q&A]
매크로 잘 쓰고 있습니다 개발자님.

그런데 카카오맵만 켜면 바로 네이버 예약페이지로 창이 이동합니다.
잔여백신이 뜨지 않았는데도요 ㅜㅜ
코드 여러번 바꿔봤지만 코드에 따라 이동하는 병원만 바뀔 뿐 계속 이동하네요..

**Comment by changdoc** [comment]
아스트라제네카가 잔여 백신이 늘면서 1.1.4버전까지는 무조건 이동 중일텐데요.
최신 버전에서 확인 부탁드립니다. https://github.com/changdoc/naver-vaccine-macro/releases/tag/v1.1.6
